pkgs/development/python-modules/indexed_gzip/default.nix

```
@@ -0,0 +1,30 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pkgs
```

Referencing pkgs directly is highly discouraged. It makes overriding an individual package very difficult to do as you would need to create a new top-level package set to then override a dependency. Instead, please list each dependency individiually, and if pulling from another package set (e.g. `python3Packages.docker` vs `pkgs.docker`), please alter the `callPackage` to reference the dependency e.g. `python3Packages.callPackage ./<package>.nix { inherit (pkgs) docker; };`
```suggestion
, zlib
```